src/App.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import TodoDefault from './TodoDefault';
+import TodoForm from './TodoForm';
+import TodoList from './TodoList';
+import TodoTitle from './TodoTitle';
+
+function App() {
+  const [todoList, setTodoList] = useState([]);
+  const [todoInput, setTodoInput] = useState('');
+  const handleSubmit = (e) => {
```

Comment:
변수를 할당한 부분과, 함수 부분은 빈줄을 넣어주시면 가독성이 더 좋아집니다 :)
```suggestion
  const [todoInput, setTodoInput] = useState('');
  
  const handleSubmit = (e) => {
```